exporters/ostream/include/opentelemetry/exporters/ostream/span_exporter.h

```diff
@@ -100,7 +100,7 @@ class OStreamSpanExporter final : public sdktrace::SpanExporter
 #if __cplusplus < 201402L
     nostd::visit(OwnedAttributeValueVisitor(*this), value);
 #else
-    nostd::visit([this](auto &&arg) { print_value(arg); }, value);
+    nostd::visit([this](auto &&arg) { this->print_value(arg); }, value);
```

Comment:
Add an comment that `this->` is required for GCC 5.4.0? In case we do clean this up later.

Reply:
thanks. comment added to code